autorest/codegen/templates/setup.py.jinja2

```diff
@@ -16,7 +80,13 @@ VERSION = "{{ code_model.options.get('package_version', '0.0.0') }}"
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-REQUIRES = ["msrest>=0.6.21", "azure-core<2.0.0,>=1.20.1"{{ azure_mgmt_core_import }}]
+REQUIRES = [
```

Comment:
even though we generate the `install_requires` different for basic-setup-py and our new package mode setup.py, the actual result is the same, so I think we can switch the basic setup py away from having a requires constant that we pass to `install_requires`. That way, we shouldn't need a separate section for "basic-setup-py", instead if it's not basic setup py, we add more things